site/assets/scss/_component-examples.scss

```diff
@@ -217,6 +217,42 @@
   }
 }
 
+.bd-example-inset-utils {
+  display: flex;
+  flex-wrap: wrap;
+
+  span {
+    position: absolute;
+    display: block;
+  }
+
+  div {
+    position: relative;
```

Comment:
It'd be better to use our `position` utilities in those examples, since they're likely to be used alongside `inset` ones.

Reply:
done!

Reply:
You need to remove the declaration here, since you used the utilities.

Reply:
Sorry my bad. I forget send my changes